asclepius/asclepius/main.py

```diff
@@ -69,12 +68,23 @@ def ddis():
     except ValueError as e:
         raise InvalidUsage(str(e))
 
-    return jsonify(to_labelled(dinto_res))
+    return jsonify(dinto_res)
 
 
 @app.route("/ping")
 def ping():
-    return '', 204
+    DUMMY_REQUEST = 'SELECT * WHERE {?s ?p ?o} LIMIT 0'
+    DUMMY_RESPONSE = {"head": {"vars": ["s", "p", "o"]},
+                      "results": {"bindings": []}}
+    try:
+        resp = requests.post(dinto.SPARQL_ENDPOINT, data={'query': DUMMY_REQUEST})
+    except requests.exceptions.ConnectionError:
```

Comment:
`RequestException` is the superclass of all the exceptions that the requests library will throw so probably better to catch that